test/utils.sh

```diff
@@ -157,18 +157,24 @@ parse_test_output() {
   fi
 }
 
-# the function will be used by the tekton tasks of build-definitions
+# The function will be used by the tekton tasks of build-definitions
+# It need tekton result path as parameter
 handle_error()
 {
-  # The tekton task result path
-  TEST_RESULT_PATH=$1
-  if [ -z "$TEST_RESULT_PATH" ]; then
-    echo "Missing parameter TEST_RESULT_PATH" >&2
-    exit 2
-  fi
+  exit_code=$?
+  if [ "${exit_code}" -ne 0 ]; then
+    # The tekton task result path
+    TEST_OUTPUT_PATH=$1
+    if [ -z "$TEST_OUTPUT_PATH" ]; then
+      echo "Missing parameter TEST_OUTPUT_PATH" >&2
+      exit 2
```

Comment:
Just wondering - would there be any benefit to returning the `${exit_code}` here instead of `2`?

Reply:
2 usually means wrong parameter, so it's case of this function when param is missing